roles/glance/templates/etc/glance/glance-swift-store.conf

```diff
@@ -0,0 +1,11 @@
+[ref1]
+{% if (glance.store_smart|bool and swift.enabled|bool) or
+      glance.store_swift|bool  %}
```

Comment:
I didn't want to "reveal" swift service password in case service is not enabled... We write all glance templates right now using fileglob.. Ideally we only write the swift-store one if Swift was enabled but then that would be different .. Thoughts?